components/02-elements/tab/tab.js

```diff
@@ -1,14 +1,14 @@
-'use strict'
+'use strict';
 
 const tab                = document.querySelectorAll('.tab__title'),
       content            = document.querySelectorAll('.tab__content'),
       activeTitleClass   = 'tab__title--active',
       activeContentClass = 'tab__content--active';
 
-
 tab.forEach(key => key.addEventListener('click', event => {
-  const self = event.target,
-        tabId = self.dataset.tab;
+  let self = event.currentTarget;
```

Comment:
Now it can be `const` and you can combine all variables together.

Reply:
Done